### PR TITLE
Zhang fix 01

### LINKED ISSUE #23: Zhang Figures
There are a few figures that are potentially not included in the Zhang articles. Additionally, the caption should likely be removed from Figure 3 in the image (cropping the figure as the caption is in the text!).

@taylorgibson can you let us know what you would like to do?

image

image

### PR DESCRIPTION
**Pull request by taylorgibson**
This should fix #23

### CODE REVIEW

**Review by fwkoch — COMMENTED** [review]
@taylorgibson The text changes look great, but the figures are missing from the `images` folder.

**Comment by taylorgibson** [comment]
@fwkoch Good catch, not sure how that happened, but the latest commit should fix that!

File: papers/zhang/article.md [hunk 1]
```diff
@@ -65,11 +65,29 @@ Asterias vulgaris were also placed in the organismal chamber to observe rates of
...
+
+```{figure} images/figure_5.png
+:label: average_dissolved_oxygen
+
+Comparison of average basal rate across all included trials and dissolved oxygen consumption from the control groups exhibited no statistically significant difference (p-value of 0.411203 on a 95% confidence interval). The injection process was then determined to not yield a significant effect on dissolved oxygen consumption rates. Basal rate measurements can be compared with results from the experimental groups.
+```
+
+The quantitative effects of exposure of ArSK/CCK1 on the dissolved oxygen consumption of A. vulgaris was observed by averaging the rate values from all organisms and trials considered (see @average_rate_values). This average (-0.0002312072727 mg/L/s) was compared to the average dissolved oxygen consumption rate for the basal rate group with a one-tailed t-test. A p-value greater than 0.05 was achieved, indicating that the average rate for the ArSK/CCK1 injection group was significantly lower than that of the basal rate. 
+
+```{figure} images/figure_6.png
```
Comment (rowanc1) [inline papers/zhang/article.md:86]:
```suggestion
```{figure} images/figure_06.png
```

File: papers/zhang/article.md [hunk 2]
```diff
@@ -65,11 +65,29 @@ Asterias vulgaris were also placed in the organismal chamber to observe rates of
...
+Dissolved oxygen consumption rates for all tested organism diameters with data from basal rate, control, and experimental groups.
+```
+
+The average change in dissolved oxygen consumption was taken across all organisms and averaged to achieve a value for comparison. To establish the validity of the H{sub}`2`O injection as a control, the average change in dissolved oxygen consumption for the basal rate and H{sub}`2`O injection measurements were compared. A one-tailed T-test was performed for the two means to determine if the average dissolved oxygen consumption rate for the basal rates (-0.0008728646154 mg/L/s) was significantly greater than the average rate for the H{sub}`2`O injection (-0.0008281736364 mg/L/s), as shown in @average_dissolved_oxygen. A p-value less than 0.05 was acquired, thus demonstrating that the difference between the two values was not significant and factors such as the injection process or addition of fluid into the coelomic cavity did not have a significant impact on the results obtained.
+
+```{figure} images/figure_5.png
```
Comment (rowanc1) [inline papers/zhang/article.md:78]:
```suggestion
```{figure} images/figure_05.png
```

File: papers/zhang/article.md [hunk 3]
```diff
@@ -65,11 +65,29 @@ Asterias vulgaris were also placed in the organismal chamber to observe rates of
...
 
-The average change in dissolved oxygen consumption was taken across all organisms and averaged to achieve a value for comparison. To establish the validity of the H{sub}`2`O injection as a control, the average change in dissolved oxygen consumption for the basal rate and H{sub}`2`O injection measurements were compared. A one-tailed T-test was performed for the two means to determine if the average dissolved oxygen consumption rate for the basal rates (-0.0008728646154 mg/L/s) was significantly greater than the average rate for the H{sub}`2`O injection (-0.0008281736364 mg/L/s), as shown in Figure 5. A p-value less than 0.05 was acquired, thus demonstrating that the difference between the two values was not significant and factors such as the injection process or addition of fluid into the coelomic cavity did not have a significant impact on the results obtained.
+```{figure} images/figure_4.png
```
Comment (rowanc1) [inline papers/zhang/article.md:70]:
```suggestion
```{figure} images/figure_04.png
```

**Review by rowanc1 — COMMENTED** [review]
Fixing the image pointers!

**Comment by rowanc1** [comment]
@taylorgibson I am going to take a quick pass at cropping the images.

**Comment by taylorgibson** [comment]
@rowanc1 Sounds good, thanks!

**Pull request merged**